src/numbers/Request.java

```diff
@@ -0,0 +1,58 @@
+package numbers;
+
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Request {
+
+    public static final String INSTRUCTIONS_PRINT_REQUEST = "";
+    public static final String PROGRAM_TERMINATE_REQUEST = "0";
+
+    final String startNumberSection;
+    final String numbersAmountSection;
+    final String numbersPropertiesSection;
+
+    public Request(final String userInput) {
+        final Scanner scanner = new Scanner(userInput).useDelimiter(" ");
```

Comment:
What if userInput = null?

Reply:
I didn't thought about it... Solution is correct according to "Hyperskill" tests, so I decided not to complicate it with validation checks.